### PR TITLE
feat: Hanoi - Add new compose builder options for 1.3.1

### PR DESCRIPTION
**Pull request by lenny-intel**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/edgexfoundry/developer-scripts/blob/master/.github/Contributing.md.

## What is the current behavior?

Hanoi Compose Builder doesn't have some new options that users requesting
device-grove YML missing `devices` section

## What is the new behavior?
Added these options:
- mqtt-bus (renamed from mqtt to be more explicit)
- mqtt-broker (new)
- asc-http (new)
- asc-mqtt (new)
- modbus-sim (new)

Aslo added this to device-grove yaml
```
devices:
         - /dev/i2c-1
```

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Specific Instructions
Are there any specific instructions or things that should be known prior to reviewing?

## Other information

### CODE REVIEW

File: compose-builder/README.md [hunk 1]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    ds-grove:    Generates compose file with device-grove included (valid only with arm64 option)
+    ds-modbus:   Generates compose file with device-modbus included
+    ds-mqtt:     Generates compose file with device-mqtt included
+    ds-random:   Generates compose file with device-random included
+    ds-rest:     Generates compose file with device-rest included
+    ds-snmp:     Generates compose file with device-snmp included
+    ds-virtual:  Generates compose file with device-virtual included
+    modbus-sim:  Generates compose file with ModBus simulator included
+    asc-http:    Generates compose file with App Service HTTP Export included
+    asc-http-s:  Generates compose file with App Service HTTP Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:126]: should this be there? remove it if not

File: compose-builder/README.md [hunk 2]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    asc-mqtt:    Generates compose file with App Service MQTT Export included
+    asc-mqtt-s:  Generates compose file with App Service MQTT Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:128]: should this be there? remove it if not

File: compose-builder/README.md [hunk 3]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    ds-camera:   Runs with device-camera included
+    ds-grove:    Runs with device-grove included (valid only with arm64 option)
+    ds-mqtt:     Runs with device-mqtt included
+    ds-random:   Runs with device-random included
+    ds-rest:     Runs with device-rest included
+    ds-snmp:     Runs with device-snmp included
+    ds-virtual:  Runs device-virtual included
+    modbus-sim:  Runs with ModBus simulator included
+    asc-http:    Runs with App Service HTTP Export included
+    asc-http-s:  Runs with App Service HTTP Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:154]: should this be there? remove it if not

File: compose-builder/README.md [hunk 4]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    asc-mqtt:    Runs with App Service MQTT Export included
+    asc-mqtt-s:  Runs with App Service MQTT Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:156]: should this be there? remove it if not

File: compose-builder/README.md [hunk 5]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    ds-grove:    Pull includes device-grove (valid only with arm64 option)
+    ds-modbus:   Pull includes device-modbus 
+    ds-mqtt:     Pull includes device-mqtt
+    ds-random:   Pull includes device-random
+    ds-rest:     Pull includes device-rest
+    ds-snmp:     Pull includes device-snmp
+    ds-virtual:  Pull includes device-virtual
+    modbus-sim:  Pull includes ModBus simulator
+    asc-http:    Pull includes App Service HTTP Export
+    asc-http-s:  Pull includes App Service HTTP Export Secrets
```
Comment (jim-wang-intel) [inline compose-builder/README.md:180]: should this be there? remove it if not

File: compose-builder/README.md [hunk 6]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    asc-mqtt:    Pull includes App Service MQTT Export
+    asc-mqtt-s:  Pull includes App Service MQTT Export Secrets
```
Comment (jim-wang-intel) [inline compose-builder/README.md:182]: should this be there? remove it if not

File: compose-builder/README.md [hunk 7]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    ds-camera:   Generates compose file with device-camera included
+    ds-grove:    Generates compose file with device-grove included (valid only with arm64 option)
+    ds-mqtt:     Generates compose file with device-mqtt included
+    ds-random:   Generates compose file with device-random included
+    ds-rest:     Generates compose file with device-rest included
+    ds-snmp:     Generates compose file with device-snmp included
+    ds-virtual:  Generates compose file with device-virtual included
+    modbus-sim:  Generates compose file with ModBus simulator included
+    asc-http:    Generates compose file with App Service HTTP Export included
+    asc-http-s:  Generates compose file with App Service HTTP Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:208]: should this be there? remove it if not

File: compose-builder/README.md [hunk 8]
```diff
@@ -100,84 +108,110 @@ compose [options]
...
+    asc-mqtt:    Generates compose file with App Service MQTT Export included
+    asc-mqtt-s:  Generates compose file with App Service MQTT Export Secrets included
```
Comment (jim-wang-intel) [inline compose-builder/README.md:210]: should this be there? remove it if not

**Review by jim-wang-intel — CHANGES_REQUESTED** [review]
just minor changes in documentation if my take is ok

File: compose-builder/README.md (hunk 1 above)
Comment (lenny-intel) [reply compose-builder/README.md:126]: Opps, meant to remove. Good catch.

Comment (lenny-intel) [reply compose-builder/README.md]: Fixed & pushed.

Comment (jim-wang-intel) [reply compose-builder/README.md]: cool, that's where extra pair of eyes works :-)

**Review by jim-wang-intel — APPROVED** [review]
LGTM

**Review by jpwhitemn — APPROVED** [review]

**Pull request merged**